presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -39,14 +39,12 @@
         extends PlanNode
```

Comment:
I think there is a typo in the commit message, as suggested by @findepi 

Take a look at the end of the sentence
```
(expression, signature and mask).s
````